packages/mobile/android/app/build.gradle

```diff
@@ -283,7 +283,6 @@ dependencies {
     } else {
         implementation jscFlavor
     }
-    compile project(':react-native-video')
```

Comment:
We sure we don't need this anymore?

Reply:
yeah it was complaining, and looking at react-native-video docs, it doesn't say to do this anymore!